### PR TITLE
ci: resolve issue where protos are not updating

### LINKED ISSUE #122: Update protos in google/cloud/audit
The protos in https://github.com/googleapis/python-audit-log/tree/main/google/cloud/audit are out of date. They should be updating automatically using [owlbot.py](https://github.com/googleapis/python-audit-log/blob/main/owlbot.py) but the automation is not working as expected.

For example,
see https://github.com/googleapis/python-audit-log/blob/main/google/cloud/audit/audit_log.proto which hasn't been updated in 4 years whereas https://github.com/googleapis/googleapis/blob/master/google/cloud/audit/audit_log.proto has been updated in the last 2 years.

### PR DESCRIPTION
**Pull request by parthea**
Fixes #122 🦕

BEGIN_COMMIT_OVERRIDE
feat: Add PolicyViolation. this will only be present when access is denied due to Organization Policy [fc5be6f](https://github.com/googleapis/googleapis/commit/fc5be6f850e7989e912b40c6b79306c6dc9655bd)
feat: Add the principal field to the ServiceAccountDelegationInfo [ba89dac](https://github.com/googleapis/googleapis/commit/ba89dace27923254d96ab8339b831dc996e2112f)
feat: Update AuditLog proto to include all new changes in Audit Logging [40292fc](https://github.com/googleapis/googleapis/commit/40292fc8f271f3b8708f9c91c85d7240200893a6)
feat: Add `google/cloud/audit/bigquery_audit_metadata_pb2.py`
END_COMMIT_OVERRIDE

### CODE REVIEW

File: google/cloud/audit/audit_log.proto [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 Google LLC
+// Copyright 2022 Google LLC
```
Comment (vchudnov-g) [inline google/cloud/audit/audit_log.proto:1]: nit: Is this the right date? I would have expected keeping the original or using 2024, but maybe the protos last changed in 2022?

**Review by vchudnov-g — APPROVED** [review]
One minor suggestion and a question about copyrights. But LGTM. Thanks for taking this on!

File: google/cloud/audit/audit_log.proto (hunk 1 above)
Comment (parthea) [reply google/cloud/audit/audit_log.proto]: Fixed in https://github.com/googleapis/python-audit-log/pull/123/commits/0f26a6caa475845c0380c4b11c7477a1bba8f570

File: owlbot.py [hunk 2]
```diff
@@ -61,34 +72,14 @@
 # Generate _pb2.py files and format them
 s.shell.run(["nox", "-s", "generate_protos"], hide_output=False)
 
-# Clean up googleapis
+# Clean up the folders for dependencies which are shipped via `googleapis-common-protos`
+# We should not ship them via this repository
+shutil.rmtree('google/api')
```
[the remark this answers is not in the document]
Comment (parthea) [reply owlbot.py]: Fixed in https://github.com/googleapis/python-audit-log/pull/123/commits/4cc54fa455dd60c54ac76a56c5bec8629bf21a88

**Pull request merged**